src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/RulesListItems/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { t } from '@lingui/macro'
+import { FUNDING_CYCLE_WARNING_TEXT } from 'constants/fundingWarningText'
+import { V2V3ProjectContext } from 'contexts/v2v3/V2V3ProjectContext'
+import {
+  V2V3FundingCycle,
+  V2V3FundingCycleMetadata,
+} from 'models/v2v3/fundingCycle'
+import { useContext } from 'react'
+import { getBallotStrategyByAddress } from 'utils/v2v3/ballotStrategies'
+import { getUnsafeV2V3FundingCycleProperties } from 'utils/v2v3/fundingCycle'
+import {
+  HOLD_FEES_EXPLAINATION,
+  OWNER_MINTING_EXPLAINATION,
+  RECONFIG_RULES_EXPLAINATION,
+  TERMINAL_CONFIG_EXPLAINATION,
+} from '../../settingExplanations'
+import { FundingCycleListItem } from '../FundingCycleListItem'
+import { AllowMintingValue } from './AllowMintingValue'
+import { AllowSetTerminalsValue } from './AllowSetTerminalsValue'
+import { HoldFeesValue } from './HoldFeesValue'
+import { PausePayValue } from './PausePayValue'
+import { ReconfigStratValue } from './ReconfigStratValue'
+
+export function RulesListItems({
+  fundingCycle,
+  fundingCycleMetadata,
+  showDiffs,
+}: {
+  fundingCycle: V2V3FundingCycle
+  fundingCycleMetadata: V2V3FundingCycleMetadata
+  showDiffs?: boolean
+}) {
+  const {
+    fundingCycle: oldFundingCycle,
+    fundingCycleMetadata: oldFundingCycleMetadata,
+  } = useContext(V2V3ProjectContext)
+
+  const riskWarningText = FUNDING_CYCLE_WARNING_TEXT()
+  const unsafeFundingCycleProperties = getUnsafeV2V3FundingCycleProperties(
+    fundingCycle,
+    fundingCycleMetadata,
+  )
+
+  const ballotStrategy = getBallotStrategyByAddress(fundingCycle.ballot)
+  const oldBallotStrategy = oldFundingCycle
+    ? getBallotStrategyByAddress(oldFundingCycle.ballot)
+    : undefined
+
+  const ballotWarningText = unsafeFundingCycleProperties.noBallot
```

Comment:
nit: complex ternaries are hard to read. where possible, it might make sense to wrap them in a memo to be a bit easier to read. No biggie though, wouldnt block merging